README.md

```diff
@@ -1,27 +1,36 @@
-# Project Name
+# Portfolio project
 
-> One paragraph statement about the project.
+> Portfolio project built with semantic HTML and some css styles like display flex etc .
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+## Built With
```

Comment:
Let's add a detailed description, it doesn't have to be too long but it should be as detailed as possible. 2 - 3 paragraphs are fine. See your assignment link - https://github.com/bolah2009/curriculum-html-css/blob/main/portfolio/sneak_peek.md for details you can use to describe the project.